packages/sampler/stories/qa-stories/Scroller.js

```diff
@@ -24,6 +25,29 @@ const
 		horizontalScrollbar: ['auto', 'hidden', 'visible']
 	};
 
+class ScrollerWithResizable extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			more: false
+		};
+	}
+
+	handleClick = () => {
+		this.setState(prevState => ({more: !prevState.more}));
```

Comment:
Why don't we simplify the code above like the following way?
```
this.setState({more: !this.state.more});
```

Reply:
If do that, eslint errors occur while travis build.

Reply:
react/no-access-state-in-setstate

Reply:
It was fine before. I see. I don't mind if you fix it or not.